src/core/Database.h

```diff
@@ -73,6 +73,7 @@ class Database : public QObject
     bool save(QString* error = nullptr, bool atomic = true, bool backup = false);
     bool save(const QString& filePath, QString* error = nullptr, bool atomic = true, bool backup = false);
     bool extract(QByteArray&, QString* error = nullptr);
```

Comment:
@jsachs We should rename that function `export` at that point I think.

Reply:
I seem to remember a conversation somewhere about the naming of this function--if you think `export` is correct I can make the change as part of this PR.

Reply:
I'd go with `export`, it makes sense especially now that we have `import`

Reply:
Was this not already handled by #3278 ?

Reply:
only the CLI command was renamed, not the `Database` function

Reply:
https://en.cppreference.com/w/cpp/keyword/export

Do we want to find a way to override this, or just leave it as is?

Reply:
If it is a function name it wouldn't clash with the keyword.

Reply:
If it is a function name it wouldn't clash with the keyword.

Reply:
mmmhhh not sure. We could also call them `exportToXML` and `importFromXML` to make it more explicit.

Reply:
I think there is a clash based on the following error:
`keepassxc/src/core/Database.h:75:10: error: expected member name or ';' after declaration specifiers
    bool export(QByteArray&, QString* error = nullptr);`

Reply:
I'd argue that based on the compiler complaints, it's fine to leave the underlying methods as `extract`, given that the user interface is going to be `import` and `export`.

Reply:
Wow I take it back, apparently you cannot name a function "export" which sucks.